docs/docs/keymap-example.md

```diff
@@ -2,20 +2,34 @@
     keymap {
         compatible = "zmk,keymap";
 
-        default_layer {
-// --------------------------------------------------------------------------------------------------------------------------------------------------------------------
-// |   ESC   |    Q    |    W    |    E    |    R    |    T    |                                          |    Y    |    U    |    I    |    O    |    P    |    \    |
-// |   TAB   |    A    |    S    |    D    |    F    |    G    |                                          |    H    |    J    |    K    |    L    |    ;    |    '    |
-// |  SHIFT  |    Z    |    X    |    C    |    V    |    B    | CTRL+A  | CTRL+C  |  |  CTRL+V |  CTRL+X |    N    |    M    |    ,    |    .    |    /    |  R CTRL |
-//                               |   GUI   |   DEL   | RETURN  |  SPACE  | ESCAPE  |  |  RETURN |  SPACE  |   TAB   |   BSPC  |  R ALT  |
+        default_layer { // Layer 0
+// ----------------------------------------------
```

Comment:
Do you think it's worth getting rid of these comment blocks? They don't seem to add much in this example, at least.

Reply:
I think the comment blocks are used frequently enough in other keymaps that it's useful to have them so that new users get used to them, in a way.